ocs_ci/deployment/deployment.py

```diff
@@ -486,6 +486,7 @@ def deploy_ocs_via_operator(self, image=None):
                 create_ocs_secret(constants.MARKETPLACE_NAMESPACE)
         if not live_deployment:
             self.create_ocs_operator_source(image)
+            pass
```

Comment:
I assume this is leftover from local testing?

Reply:
oops, yes , I missed it.